clang/test/CodeGenHLSL/semantics/DispatchThreadID.hlsl

```diff
@@ -1,28 +1,25 @@
-// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -emit-llvm -finclude-default-header -disable-llvm-passes -o - %s

```

Comment:
Note: This test didn't actually have a call to `FileCheck` before, so I had to make some minor changes to get the existing CHECKs passing.